src/components/Login/Login.jsx

```diff
@@ -17,24 +22,45 @@ const Login = () => {
 
   return (
     <>
+      <CssBaseline />
```

Comment:
I believe CssBaseLine is something that should be used in the global index.jsx for MUI's CSS Reset, not here in individual components.